### PR TITLE
Remove coleads

### PR DESCRIPTION
**Pull request by maxwofford**

### CODE REVIEW

File: src/pages/apply/index.js [hunk 1]
```diff
@@ -141,7 +144,44 @@ const ApplicationCard = props => {
...
                 </span>
               )}
-              finished their leader profile
+              finished their leader profile{' '}
+              {isPoc ? (
+                <Fragment>
+                  (
+                  <A
+                    onClick={e => {
+                      if (
```
Comment (lachlanjc) [inline src/pages/apply/index.js:171]: Prefer ternary expressions to if inside JSX :)

(Update: I read this code too quickly, don't use a ternary expression here)

Comment (maxwofford) [reply src/pages/apply/index.js:171]: Is that still true inside an anonymous function?

Comment (maxwofford) [reply src/pages/apply/index.js:171]: Going to merge for the sake of time (need to get this up for an applicant ASAP), but I'll refactor it

Comment (maxwofford) [reply src/pages/apply/index.js:171]: How would you recommend writing this?

Comment (lachlanjc) [reply src/pages/apply/index.js:171]:
```js
{isPoc && ( // allows excluding the `: null` at the end
  // the <Fragment> and the ( are not needed here
  <A
    onClick={e => {
      // if you're accessing properties multiple times, destructure
      const { id: user_id, email } = profile.user
      if (
        confirm(`Are you sure you want to remove ${email} as a team member?`)
      ) {
        api
          .delete(`v1/new_club_applications/${id}/remove_user`, {
            authToken,
            data: { user_id } // yay destructuring
          })
          .then(json => {
            callback()
            alert(`${email} has been removed from the application`)
          })
      }
    }}
    style={{ cursor: 'pointer' }}
    children="remove" // I go back and forth on using children prop
  />
)}
```

Comment (lachlanjc) [reply src/pages/apply/index.js:171]: Again, building + shipping things is way more important than code style :) (but……lol)

**Pull request merged**